src/dataops.jl

```diff
@@ -0,0 +1,212 @@
+
+"""
+    dcopy(dInfo::LoadedDataInfo, newName::Symbol)
+
+Copy the dataset into a new place.
```

Comment:
new place? might be a bit misleading. do you mean: copy into a new variable, or really a new place like different worker?